addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/EnvFileScanRule.java

```diff
@@ -33,6 +35,10 @@
     private static final Pattern COMMENT_PATTERN =
             Pattern.compile("^#\\s{0,10}\\w+", Pattern.MULTILINE);
     private static final Pattern KEYVAL_PATTERN = Pattern.compile("^\\w+=\\w+", Pattern.MULTILINE);
+    private static final Map<String, String> ALERT_TAGS =
+            CommonAlertTag.toMap(
+                    CommonAlertTag.OWASP_2021_A05_SEC_MISCONFIG,
+                    CommonAlertTag.OWASP_2017_A06_SEC_MISCONFIG);
```

Comment:
Could be argued as: Authz? Access Control? Design?

Same with hidden file and trace axd.

Reply:
I went for sec misconfig for these sort of vulns in the release rules - happy to change but I think we should be consistant for classes of rules like this.